redash/query_runner/oracle.py

```diff
@@ -176,7 +187,11 @@ def run_query(self, query, user):
                 json_data = json_dumps(data)
                 connection.commit()
         except cx_Oracle.DatabaseError as err:
-            error = "Query failed. {}.".format(str(err))
+            (err_args,) = err.args
+            line_num, column_num = Oracle.calculate_line_and_column(query, err_args.offset)
+            error = "Query failed. {}. Query failed at Line {}, Column {}.".format(
```

Comment:
Maybe simplify it to something like... ?

    Query failed at line {}, column {}: {}.

That's just a rough guess without trying it though. :smile: